packages/esm-patient-vitals-app/src/vitals/vitals-biometrics-form/vitals-biometrics-input.component.tsx

```diff
@@ -68,12 +84,29 @@ const VitalsBiometricInput: React.FC<VitalsBiometricInputProps> = ({
   }
 
   return (
-    <div className={styles.inputContainer} style={{ width: textFieldWidth }}>
-      <p className={styles.vitalsBiometricInputLabel01}>{title}</p>
+    <div
+      className={`${styles.inputContainer} ${isTablet ? styles.inputInTabletView : ''} ${
+        flaggedCritical ? styles.isCriticalInput : ''
+      }`}
+      style={{ width: textFieldWidth }}
+    >
+      <div className={styles.labelAndIcons}>
+        <p className={styles.vitalsBiometricInputLabel01}>{title}</p>
+        {flaggedAbnormal ? (
+          <div title="abnormal value">
+            {interpretation === 'high' ? <span className={styles.high}></span> : null}
+            {interpretation === 'critically_high' ? <span className={styles['critically-high']}></span> : null}
+            {interpretation === 'low' ? <span className={styles.low}></span> : null}
+            {interpretation === 'critically_low' ? <span className={styles['critically-low']}></span> : null}
```

Comment:
This is probably the once case I would prefer nested ternaries to this although you could also just do:

```
<span className={styles[interpretation.replace("_", "-")]} />
```